psychopy/tests/test_all_visual/test_contains_overlaps.py

```diff
@@ -23,12 +19,12 @@
 sqrt2 = sqrt(2)
 
 points = [
-    array((0,0)),
-    array((0,unitDist)),
-    array((0,unitDist*2)),
-    array((old_div(unitDist,sqrt2),old_div(unitDist,sqrt2))),
-    array((unitDist*sqrt2,0)),
-    array((unitDist*sqrt2,unitDist*sqrt2)) ]
+    array((0, 0)),
+    array((0, unitDist)),
+    array((0, unitDist * 2)),
+    array(((unitDist / sqrt2) / (unitDist / sqrt2))),
```

Comment:
@mdcutone this is the bug that's causing the failure with shape pos in the test suite